xonsh/built_ins.py

```diff
@@ -466,6 +467,8 @@ def run_subproc(cmds, captured=False):
             stderr = builtins.__xonsh_stderr_uncaptured__
         uninew = (ix == last_cmd) and (not _capture_streams)
         alias = builtins.aliases.get(cmd[0], None)
+        if alias is None and cmd[0] in builtins.__xonsh_enabled_builtin_commands__:
+            alias = all_builtin_commands[cmd[0]]
```

Comment:
I am not sure I follow why just using aliases isn't good enough.  This seems like it creates a secondary alias system...